src/foam/lib/Outputter.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib;
+
+import foam.core.FObject;
+
+// not modelled because method with
+// name stringify is not generated
+public interface Outputter {
+  String stringify(FObject obj);
```

Comment:
Couldn't we add the output(Object) method here as well?

Reply:
Yes done.